### DISCUSSION TITLE
Incorrect dcc-id triggers unhelpful error

### DISCUSSION THREAD

**Discussion by dankolbman** [nih-cfde/published-documentation#205] [Q&A] [answered]
The following error will occur when specifying a dcc-id without a `:`:
```
This Flow has completed.
Your CFDE submission () failed to ingest into DERIVA with this error:
Error ingesting to DERIVA: not enough values to unpack (expected 2, got 1)
```
It seems that there is a `namespace, local_id = composite.split(':')` happening somewhere.
This should be updated to handle the case where arbitrary amounts of `:` may exist and also be checked in the `cfde-submit` cli.

**Comment by ACharbonneau** [comment, marked as answer]
Thanks! I'm pretty sure this is a variation on/duplicate of https://github.com/nih-cfde/published-documentation/discussions/150 but I'll copy the answer here:

This error likely means that you tried to use a DCC ID that was not recognized. DCC IDs are two parts, and you submitted only one. 

**Possible solutions:**

1. Find your correct DCC ID

Use [these instructions](https://docs.nih-cfde.org/en/latest/cfde-submit/docs/#find-your-dccs-identifier) to find your DCC ID. Be sure to use the full ID in the tool, it should look something like `cfde_registry_dcc:hmp`. IDs are space and case sensitive.

**Reply by ACharbonneau** [reply]
This has been fixed. As of 3/30/2021 the submission tool now takes DCC IDs in the form of `cfde_registry_dcc:hmp` or just `hmp`
